### DISCUSSION TITLE
Create a Project View Layout of Calendar

### DISCUSSION THREAD

**Discussion by kcgrimes** [community/community#118846] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Projects

### Body

In Projects, it would be helpful to have a View Layout of a Calendar. Roadmap is effectively a Gantt chart and is a start, but a true calendar provides yet another dynamic means of project management (that is frankly much more impactful than Roadmap). 
Ideal features include: 
Option to "zoom" to Year, Month, Week, Day
Option to select what date to view
Option to have undated issues in a list on the side visible or not
Drag & drop to change target date
Issue "cards" could be compact like in Table, or bulkier like in Board; but still having option to show custom columns (i.e. Priority) and Labels
Ability to sort multiple cards on a day as in other layouts

To consider another product, essentially the viewing functionality of Google Calendar but as a github Project View Layout. 

Similar to below repo, but directly integrated - 
https://github.com/planetscale/issues-calendar-for-github-projects

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by HigeRoman** [comment]
Would certainly like to have this view as well! my organization is very much so used to using Calendars to work on so it'd make the transition to github so much easier!

**Comment by DjakaTechnology** [comment]
Supporting this, its very useful. It would be lovely instead of switching around with google calendar.

Our team need to mark important dates and holidays, we're currently using the road map features but it's not as easy to use as having it in calendar view

**Comment by sarah-r-woodrow** [comment]
This would be really useful - our team need to keep track of important release deadlines and holidays

**Comment by fernandobatels** [comment]
I was looking for the exact same thing. So I built a lightweight bridge project to solve this.

It generates a secure iCal feed from your GitHub Projects so you can sync deadlines directly to Apple, Google, or Outlook calendars.

To address the privacy concerns common in these threads, I built it with a 100% stateless architecture.

If you want to try it out: https://dash.projectsics.com/

It’s been a lifesaver for my own workflow. Hope it helps!
